packages/data-context/src/util/migration.ts

```diff
@@ -192,22 +193,20 @@ async function findByTestingType (cwd: string, dir: string | null, testingType:
   .map((relative) => ({ relative, testingType }))
 }
 
+export interface GetRelativeSpecs {
+  before: RelativeSpecWithTestingType[]
+  after: RelativeSpecWithTestingType[]
+}
+
 export async function getSpecs (
   projectRoot: string,
-  componentDirPath: string | null,
   e2eDirPath: string | null,
-): Promise<{
-  before: RelativeSpecWithTestingType[]
-  after: RelativeSpecWithTestingType[]
-}> {
-  const [comp, e2e] = await Promise.all([
-    findByTestingType(projectRoot, componentDirPath, 'component'),
-    findByTestingType(projectRoot, e2eDirPath, 'e2e'),
-  ])
+): Promise<GetRelativeSpecs> {
+  const e2e = await findByTestingType(projectRoot, e2eDirPath, 'e2e')
```

Comment:
So we do not rename component specs anymore?

My understanding was that we wanted the rename to avoid collision with jest when moving component tests.

Reply:
We never did auto rename on component specs - only E2E. Component is **always** manual.

Basically,

- autoRename === e2e
- manualRename === CT

Reply:
https://www.figma.com/file/PAOozuCxtfetqP8fUVvg0y/Cypress-App?node-id=4345%3A32803...

Reply:
Edit, apparently this is NOT the case, rename is automated, moving is not. :face_exhaling: